code/datums/gas_mixture.dm

```diff
@@ -103,84 +121,95 @@ What are the archived variables for?
 
 
 /datum/gas_mixture/proc/react(atom/dump_location)

```

Comment:
I so much want per-gas reactions instead of giant proc crumped with everything
If call overhead is a problem why do we have fire() as separate proc then, if we are into crumping can as well crump absolutely everything together